crates/subspace-farmer/src/single_disk_farm/plotting.rs

```diff
@@ -641,9 +646,10 @@ where
         let _ = initial_plotting_finished.send(());
     }
 
-    let mut sectors_expire_at = HashMap::with_capacity(usize::from(target_sector_count));
+    let mut sectors_expire_at =
```

Comment:
Why do we need to declare the generic params here? It seems that this data type is not changed.

Reply:
It did not and IDE had no issues with it, but compiler didn't like it anymore and required explicit types. Happens sometimes when querying happens before inserts and some similar cases.